psydac/polar/dense.py

```diff
@@ -194,6 +194,10 @@ def __init__( self, V, data ):
     def space( self ):
         return self._space
 
+    @property
+    def ghost_regions_in_sync(self):
+        return True
+
```

Comment:
If you look at the `Vector` base class [here](https://github.com/pyccel/psydac/blob/3a76c8de807ea73272a2be9ffa596fa6b7d9eca7/psydac/linalg/basic.py#L38) you will see that this method is not part of the abstract interface, so it should be moved to after line 265.

Reply:
done